lib/ts/recipe/session/recipe.ts

```diff
@@ -44,12 +51,11 @@ export default class Session extends RecipeModule<unknown, unknown, unknown, any
     private eventListeners = new Set<(ctx: RecipeEventWithSessionContext) => void>();
     private redirectionHandlersFromAuthRecipes = new Map<string, (ctx: any, history: any) => Promise<void>>();
 
-    constructor(config: ConfigType) {
-        const normalizedConfig = { ...config, ...normaliseRecipeModuleConfig(config) };
-        super(normalizedConfig);
+    constructor(config: NormalisedConfigWithAppInfoAndRecipeID<ConfigType>) {
+        super(config);
 
         this.webJsRecipe = new WebJSSessionRecipe({
```

Comment:
This doesn't use the webJS Session recipe initialized during `.init` in this recipe